### ISSUE TITLE
Maximum call stack size exceeded with orga transformer on gatsbyjs 2.3.4

### ISSUE THREAD

**Issue by kepi** [xiaoxinghu/orgajs#24]
After package upgrade to `gatsbyjs` version `2.3.4` I'm getting `UNHANDLED REJECTION` when `gatsby-transformer-orga` is enabled.

```
❯ gatsby develop                                                        
success open and validate gatsby-configs — 0.050 s
success load plugins — 0.235 s           
success onPreInit — 0.188 s                                             
success initialize cache — 0.012 s
success copy gatsby files — 0.033 s
success onPreBootstrap — 0.010 s
success source and transform nodes — 0.275 s
error UNHANDLED REJECTION                                               

                                         
  RangeError: Maximum call stack size exceeded                          

  - RegExp.test 

  - date.js:93 looksLikeADate
    [igloonet.hosting]/[gatsby]/dist/schema/types/date.js:93:109
                                            
  - example-value.js:161 getType
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:161:14
                              
  - example-value.js:40 nodes.map.node
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:40:20
                                   
  - Array.map                   
                                            
  - example-value.js:38 Array.from.reduce
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:38:27
                                       
  - Array.reduce                     
                               
  - example-value.js:37 getExampleObject   
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:37:44                              

  - example-value.js:96 Array.from.reduce
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:96:29
                        
  - Array.reduce
                  
  - example-value.js:37 getExampleObject
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:37:44

  - example-value.js:96 Array.from.reduce                                 
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:96:29

  - Array.reduce
                                                 
  - example-value.js:37 getExampleObject                                                                                              
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:37:44
                                     
  - example-value.js:96 Array.from.reduce
    [igloonet.hosting]/[gatsby]/dist/schema/infer/example-value.js:96:29

  - Array.reduce
```

**Comment by xiaoxinghu** [comment]
Tried this, confirmed, will try to figure out the reason. Thanks!

**Comment by xiaoxinghu** [comment]
fixed in 1.1.1, please try it, feel free to reopen if any more issues.

**Comment by kepi** [comment]
Thanks! Working without any problem.
